src/plugins/newsfeed/public/plugin.test.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { take } from 'rxjs/operators';
+import { coreMock } from '../../../core/public/mocks';
+import { NewsfeedPublicPlugin } from './plugin';
+import { NewsfeedApiEndpoint } from './lib/api';
+
+describe('Newsfeed plugin', () => {
+  let plugin: NewsfeedPublicPlugin;
+
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  beforeEach(() => {
+    plugin = new NewsfeedPublicPlugin(coreMock.createPluginInitializerContext());
+  });
+
+  describe('#start', () => {
+    beforeEach(() => {
+      plugin.setup(coreMock.createSetup());
+    });
+
+    describe('screenshot mode', () => {
+      it('mounts chrome nav UI', () => {
+        const coreStart = coreMock.createStart();
+        plugin.start(coreStart, { screenshotMode: { isScreenshotMode: () => false } });
+        expect(coreStart.chrome.navControls.registerRight).toHaveBeenCalled();
+      });
```

Comment:
NIT: `describe` block description is misleading, I would split between `when screenshot mode is enabled` and `when screenshot mode is disabled`